wordpress-mobile/WordPress-Android
# Contact Support: fixes for feedback from CfT

Fixes #19168 

<img width=320 src="https://github.com/wordpress-mobile/WordPress-Android/assets/990349/f236cb59-5c14-4872-8720-3b566fb5e72b" />


To test:

- Launch JP app
- `Go` to Contact support (Me -> Help -> Contact support)
- `Ensure` feature flag is enabled (Me -> Debug settings -> `contact_support_chatbot`)
- `Verify` Support Bot is launched
- `Verify` it shows the new First message
- `Verify` it shows only three suggestions

>**Note**
> Three questions have been commented out for now, just in case if we need to bring them back.
> Will clean-up before merging

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
Existing tests

PR submission checklist:


UI Changes testing checklist:
